ancar/portfolio/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns map data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}] */
+@WebServlet("/map-data")
+public class MapServlet extends HttpServlet {
+
+    private class elementMap {
```

Comment:
Class names should be in CamelCase, starting with capital letter, 
so this one should be ElementMap (or MapElement).

Also, this internal class doesn't need to know anything about it's
parent class, so it could be defined as 
```java
private static class MapElement {
...
}
```